src/utils/DataProcessingUtils.js

```diff
@@ -184,10 +196,10 @@ function processEntityValue(key :string, value :any, mappers :Map) :any {
     if (isValidDataPrimitiveArray(processedValue)) {
       return processedValue;
     }
-    if (Immutable.isList(processedValue)) {
+    if (List.isList(processedValue)) {
```

Comment:
What's the difference between `List.isList` and `Immutable.isList`?

Reply:
is the benefit of doing `List.isList` instead that you don't have to import the entire `Immutable` collection?

Reply:
More familiar to read (ie. Array.isArray). It's shorter. I can avoid importing  default `Immutable`